docs/build/guides/dapps/soroban-contract-init-template.mdx

```diff
@@ -1,442 +1,282 @@
 
-Finally, to interact with the contract we run the following code:
+Alright. Go ahead and open the app in your browser. Solid runs its dev server on `http://localhost:3000`. Does it load? It should! We didn't actually use the new `src/contracts/*` stuff in the app itself!
 
-```javascript
-const { result } = await contract.hello({ to: receiver });
-```
+### 6. Wait. Should the template actually use the contracts?
 
-### Modifying the App component
-
-To put it all together, modify the `App.jsx` component in the `src` directory by replacing its content with the code below.
-
-```javascript
-import { Client, networks } from "../packages/hello_world";
-import { useState } from "react";
-
-function App() {
-  const contract = new Client({
-    ...networks.testnet,
-    rpcUrl: "https://soroban-testnet.stellar.org:443",
-  });
-  console.log(contract);
-
-  const [target, setTarget] = useState("");
-  let receiver;
-
-  const getResult = async () => {
-    const { result } = await contract.hello({ to: receiver });
-    console.log(result);
-    setTarget(result);
-  };
-
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    getResult();
-  };
-
-  return (
-    <>
-      <h1>Response: {target}</h1>
-      <form onSubmit={handleSubmit}>
-        <input type="text" onChange={(e) => (receiver = e.target.value)} />
-        <input type="submit" value="Say hello" />
-      </form>
-    </>
-  );
-}
+Good question!
 
-export default App;
-```
+You know that the `hello_world` contract will always be there. So you could demonstrate to your users how to import and use it in an app file (like `src/App.tsx`, in the Solid template). But remember: no other contracts are guaranteed to be there! So don't commit any changes that import and use something like `increment`!
 
-Now run `npm run dev` to run the project and play with the deployed smart contract from your frontend app.
+Here's a way you could demonstrate how to import and use the `hello_world` contract in the Solid template's `App.tsx`:
 
-## Customization
+```diff
++import greeter from "./contracts/hello_world";
+
+ const App: Component = () => {
+   return (
+     <div class={styles.App}>
+       <header class={styles.header}>
++        <form
++          onSubmit={async (e) => {
++            e.preventDefault();
++            const { result } = await greeter.hello({
++              to: e.currentTarget.toWhom.value,
++            });
++            alert(result);
++          }}
++        >
++          <label>
++            Who you gonna call? &nbsp;
++            <input name="toWhom" value="World" /> &nbsp;
```

Comment:
I wanted to make this say "ghost busters!" but we need to wait for a fix to the CLI to make that possible

- https://github.com/stellar/stellar-cli/pull/1577